webservice/routes.go

```diff
@@ -52,7 +52,7 @@ var routes = Routes{
 	Route{
 		"DeletePlayer",
 		"DELETE",
-		"/players/{id}",
+		"/deletePlayer/{id}",
```

Comment:
We should allow the delete command to be sent via rest instead of being explicit here.